lib/srfi/26/26#.scm

```diff
@@ -0,0 +1,9 @@
+(##namespace ("srfi/26#" cut cute))
+
+(define-macro (cut . list<argument-or-slot>)
```

Comment:
The use of `define-macro` should be avoided because the location information of the `cut` arguments will be lost.  Using `##define-syntax` avoids this issue.  Please see `lib/_test/_test#.scm` and `lib/_test/test-expand.scm` for an example of how this should be done.

Reply:
I ended up using the syntax->list form instead of the ##deconstruct-call forms I saw in _test, I think this is solved now.